rapids-4-spark-tools/README.md

```diff
@@ -25,30 +25,30 @@ org.apache.spark.sql.rapids.tool.profiling.ProfileMain.main(Array("/path/to/even
 
 ## How to compile and use from command-line
 1. `mvn clean package`
-2. `cd $SPARK_HOME (Download Apache Spark if reequired)`
-3. `./bin/spark-submit --class org.apache.spark.sql.rapids.tool.profiling.ProfileMain  <Spark-Rapids-Repo>/workload_profiling/target/rapids-4-spark-tools-<version>.jar /path/to/eventlog1`
+2. `cd $SPARK_HOME (Download Apache Spark if required)`
+3. `./bin/spark-submit --class org.apache.spark.sql.rapids.tool.profiling.ProfileMain  <Spark-Rapids-Repo>/rapids-4-spark-tools/target/rapids-4-spark-tools-<version>.jar /path/to/eventlog1`
```

Comment:
I missed this in the earlier PR, but the directory name should just be `tools` rather than `rapids-4-spark-tools`, for consistency with the other modules in the project. The module name should stay as `rapids-4-spark-tools` though.

We can change this in a separate PR though, once this one is merged, IMO.

Reply:
yep, agree, I missed that earlier as well, lets change in followup

Reply:
Sure, Will change in followup PR.